Dockerfile

```diff
@@ -34,13 +34,24 @@ RUN make install
 
 ENV HOME="/home/ubuntu"
 ENV JASMINEGRAPH_HOME="/home/ubuntu/software/jasminegraph"
+
 RUN mkdir /home/ubuntu/software/jasminegraph
 WORKDIR /home/ubuntu/software/jasminegraph
+
 RUN pip install tensorflow==2.5.3
 RUN pip install -U scikit-learn
-COPY . .
+COPY ./GraphSAGE ./GraphSAGE
 RUN pip install -r ./GraphSAGE/requirements
 
+COPY ./conf ./conf
+COPY ./src ./src
+COPY ./src_python ./src_python
+COPY ./build.sh ./build.sh
+COPY ./CMakeLists.txt ./CMakeLists.txt
+COPY ./main.cpp ./main.cpp
```

Comment:
Why do we need to copy C++ sources to docker image? Can't we just copy the executable and library dependencies?

Reply:
We need C++ sources as they are compiled during Docker build

Reply:
Ok